docs/guide/counter/cmd/countercli/commands/counter.go

```diff
@@ -55,6 +58,10 @@ func counterTx(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	//get the nonce accounts
+	var addr []byte
```

Comment:
GetSigner in basecli/commands/cmd go should be exposed and used here. 

I had all the same issues with sendtx, just take a look there....